dolphinscheduler-task-plugin/dolphinscheduler-task-aliyunserverlessspark/src/main/java/org/apache/dolphinscheduler/plugin/task/aliyunserverlessspark/AliyunServerlessSparkTask.java

```diff
@@ -0,0 +1,244 @@
+                Thread.sleep(10 * 1000L);
+            }
+
+            setExitStatusCode(mapFinalStateToExitCode(currentState));
+
+        } catch (Exception e) {
+            log.error("Serverless spark job failed!", e);
+            throw new AliyunServerlessSparkTaskException("Serverless spark job failed!");
+        }
+    }
+
+    @Override
+    public void submitApplication() throws TaskException {
+
+    }
+
+    @Override
+    public void trackApplicationStatus() throws TaskException {
+
+    }
+
+    protected int mapFinalStateToExitCode(RunState state) {
+        switch (state) {
+            case Success:
+                return TaskConstants.EXIT_CODE_SUCCESS;
+            case Failed:
+                return TaskConstants.EXIT_CODE_KILL;
+            default:
+                return TaskConstants.EXIT_CODE_FAILURE;
+        }
+    }
+
+    @Override
+    public AbstractParameters getParameters() {
+        return aliyunServerlessSparkParameters;
+    }
+
+    @Override
+    public void cancelApplication() throws TaskException {
+        CancelJobRunRequest cancelJobRunRequest = buildCancelJobRunRequest();
+        try {
+            aliyunServerlessSparkClient.cancelJobRun(aliyunServerlessSparkParameters.getWorkspaceId(), jobRunId,
+                    cancelJobRunRequest);
+        } catch (Exception e) {
+            log.error("Failed to cancel serverless spark job run", e);
+        }
+    }
+
+    @Override
+    public List<String> getApplicationIds() throws TaskException {
+        return Collections.emptyList();
+    }
+
+    protected Client buildAliyunServerlessSparkClient(String accessKeyId, String accessKeySecret,
+                                                      String regionId, String endpoint) throws Exception {
+        if (StringUtils.isEmpty(endpoint)) {
+            endpoint = String.format(AliyunServerlessSparkConstants.ENDPOINT_TEMPLATE, regionId);
+        }
+
+        Config config = new Config()
+                .setEndpoint(endpoint)
+                .setAccessKeyId(accessKeyId)
+                .setAccessKeySecret(accessKeySecret);
+        return new Client(config);
+    }
+
+    protected StartJobRunRequest buildStartJobRunRequest(AliyunServerlessSparkParameters aliyunServerlessSparkParameters) {
+        StartJobRunRequest startJobRunRequest = new StartJobRunRequest();
+        startJobRunRequest.setRegionId(regionId);
+        startJobRunRequest.setResourceQueueId(aliyunServerlessSparkParameters.getResourceQueueId());
+        startJobRunRequest.setCodeType(aliyunServerlessSparkParameters.getCodeType());
+        startJobRunRequest.setName(aliyunServerlessSparkParameters.getJobName());
+        String engineReleaseVersion = aliyunServerlessSparkParameters.getEngineReleaseVersion();
+        engineReleaseVersion =
+                StringUtils.isEmpty(engineReleaseVersion) ? "esr-2.1-native (Spark 3.3.1, Scala 2.12, Native Runtime)"
```

Comment:
Can we extract this `esr-2.1-native (Spark 3.3.1, Scala 2.12, Native Runtime)` to constants? eg. `AliyunServerlessSparkTaskConstants`